build.gradle

```diff
@@ -585,6 +585,7 @@ dependencies {
     // used for serializing/deserializing rcf models.
     compile group: 'io.protostuff', name: 'protostuff-core', version: '1.7.4'
     compile group: 'io.protostuff', name: 'protostuff-runtime', version: '1.7.4'
+    compileOnly group: 'org.apache.commons', name: 'commons-lang3', version: '3.12.0'
```

Comment:
Thanks @ylwu-amzn for the change, could link the changes from Job-Scheduler 1.3.0 to this PR?

Reply:
Checked latest few changes in job-scheduler, don't see any related with commons-lang3. Not sure why.